### ISSUE TITLE
Adicionar owners à organização

### ISSUE THREAD

**Issue by alanhoff** [nodebr/nodebr#58]
@rafadev7 você poderia adicionar eu e/ou o @thebergamo como owners da organização NodeBR? Preciso fazer uma limpa aqui. Ex: https://github.com/nodebr/huehuebrbr

**Comment by thebergamo** [comment]
@alanhoff vamos sentar uns 20 minutos pra falar sobre isso. Eu havia te mandando msg disso no slack.

**Comment by rafaeldaime** [comment]
Sim, sem problemas. Estou em viagem, quando tiver na frente de um
computador eu faço.

Em Sex, 11 de nov de 2016 19:57, Marcos Vinicius Bérgamo <
notifications@github.com> escreveu:

**Comment by alanhoff** [comment]
@rafadev7 show, sem pressa e curta a viagem :-)

**Comment by alanhoff** [comment]
@rafadev7, novidades? :-)
